dynalab/readme.md

```diff
@@ -0,0 +1,126 @@
+# Dynalab 2.0
+
+## What is Dynalab 2.0?
+
+Dynalab 2.0 is the tool that you are going to use when you want to upload a model to a challenge on dynabench. This tool helps you to get the logic of your model and deploy it in a way the owner of the challenge can evaluate and score the performance of your algorithm.
```

Comment:
Dynalab 2.0 is the tool used to upload a model in Dynabench challenges. This tool allows to deploy a model and display it so that the model can be evaluated in the different datasets of the challenge.